SurveyScreen.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { WebView } from 'react-native-webview';
+
+{/*
+    The below code, lines 21-63 utilized chatGPT:
```

Comment:
I included this just to explain why we had to consult chatGPT when trying to get the survey in run in React Native.